documentation/docs/25-build-and-deploy/40-adapter-node.md

```diff
@@ -186,6 +186,22 @@ By default `adapter-node` gracefully shuts down the HTTP server when a `SIGTERM`
 
 > If you want to customize this behaviour you can use a [custom server](#custom-server).
 
+You can listen to the `sveltekit:shutdown` event which is emitted after the HTTP server has finished shutting down. Unlike Node's `exit` event, the `sveltekit:shutdown` event is always emitted when a shutdown is triggered and also supports handling asynchronous code.
```

Comment:
> Unlike Node's `exit` event, the `sveltekit:shutdown` event is always emitted when a shutdown is triggered

What do you mean by this? When isn't `exit` emitted? I don't see anything about it only happening sometimes in the Node docs

> and also supports handling asynchronous code.

maybe we could say "asynchronous operations" instead of "asynchronous code" to match the wording of the Node docs

Reply:
`exit` is emitted automatically when Node has no work left to do. But there might be the scenario where the SvelteKit server shuts down because we listen for a `SIGINT` but the Node app still doesn't close the user has a Mongo DB connection in the background that prevents it. That's what's happening in the comment I linked to at the beginning.

Reply:
`exit` is emitted automatically when Node has no work left to do. But there might be the scenario where the SvelteKit server shuts down because we listen for a `SIGINT` but the Node app still doesn't close because the user has a Mongo DB connection in the background that prevents it. That's what's happening in the comment I linked to at the beginning. In that case, even though the SvelteKit server closed, a `exit` event is not emitted. And if we manually emit the `exit` event after the server closed using `process.exit()` we can't run any async cleanup operations because `process.exit()` forcefully closes the app as quick as possible.